From 09219eb747cac0bc512359d3cda5f9520b0120ad Mon Sep 17 00:00:00 2001
From: Peter Wu <lekensteyn@gmail.com>
Date: Wed, 4 Sep 2013 17:42:23 +0200
Subject: [PATCH 13/24] Revert "Makefile: make user append to CFLAGS instead of
 overwriting it"

This reverts commit 98f9432088604a9a9336d6b8458bd5e010832437. This was a
lazy trick from me to allow adding `-fsanitize=address` easier, but
apparantly not all users are so happy with it.

Conflicts:
	Makefile
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-override CFLAGS := -g -O2 -Wall -Wextra -D_FORTIFY_SOURCE=2 -fstack-protector --param ssp-buffer-size=4 $(CFLAGS)
+CFLAGS ?= -g -O2 -Wall -Wextra -D_FORTIFY_SOURCE=2 -fstack-protector --param ssp-buffer-size=4
 # for install-home
 BINDIR ?= $(HOME)/bin
 
